MAUI/Scheduler/migration.md

```diff
@@ -1190,6 +1190,11 @@ this.Content = scheduler;
 <td>{{'[ResourceIdsMapping]()'| markdownify }} (From DaysView, TimelineView)</td>
 <td>Gets or sets the resource ids mapping to map the custom appointment property with the schedule appointments resource ids property.</td></tr>
 
+<tr>
+<td>{{'[TextColorMapping](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.ScheduleAppointmentMapping.html#Syncfusion_SfSchedule_XForms_ScheduleAppointmentMapping_TextColorMapping)'| markdownify }}</td>
+<td>TextColorMapping</td>
+<td>Gets or sets the TextColor Mapping to map the custom appointment property with schedule appointments text color property.</td></tr>
```

Comment:
<td>Gets or sets the TextColor Mapping to map the custom appointment property with the scheduled appointment's text color property.</td></tr>

Reply:
content changed